src/rprblender/engine/viewport_engine.py

```diff
@@ -516,6 +516,7 @@ def sync(self, context, depsgraph):
         self.rpr_context.set_parameter(pyrpr.CONTEXT_ITERATIONS, 1)
         scene.rpr.export_render_mode(self.rpr_context)
         scene.rpr.export_ray_depth(self.rpr_context)
+        self.rpr_context.texture_compression = scene.rpr.texture_compression
```

Comment:
Wouldn't it be better to set the compression option inside the scene.rpr.init_rpr_context call?

Reply:
this setting could change after the context is created